backend/common/src/main/java/ai/verta/modeldb/common/authservice/RoleServiceUtils.java

```diff
@@ -30,6 +32,16 @@ public RoleServiceUtils(AuthService authService, Integer timeout, UAC uac) {
     this.uac = uac;
   }
 
+  public static boolean checkAllResourceAllowed(Collection<Resources> resources) {
+    return resources.stream().allMatch(Resources::getAllResourceIds) && !resources.isEmpty();
```

Comment:
might as well put the `!resources.isEmpty()` first, since that's most likely cheaper than creating a stream (at the very least, it'll allocate one less object).

Reply:
changed the order